core/templates/services/exploration-html-formatter.service.ts

```diff
@@ -145,20 +146,14 @@ export class ExplorationHtmlFormatterService {
     // set as attribute keys (like '[' or ']'). So when interaction is migrated
     // we first test whether the other parts of the attribute can be added
     // (code above) and then we add the attribute using string concatenation.
-    if (
-      this.migratedInteractions.indexOf(interactionId) >= 0 &&
-      (lastAnswerPropValue !== null || savedSolution === 'savedMemento()')
-    ) {
-      let interactionHtml = element.outerHTML;
-      const tagEnd = '></oppia-interactive-' + htmlInteractionId + '>';
-      let interactionHtmlWithoutEnd = interactionHtml.replace(tagEnd, '');
-      if (savedSolution === 'savedMemento()') {
-        interactionHtmlWithoutEnd += ` [saved-solution]="${savedSolution}"`;
-      }
-      interactionHtmlWithoutEnd += ` [last-answer]="${lastAnswerPropValue}"`;
-      return interactionHtmlWithoutEnd + tagEnd;
+    let interactionHtml = element.outerHTML;
+    const tagEnd = '></oppia-interactive-' + htmlInteractionId + '>';
+    let interactionHtmlWithoutEnd = interactionHtml.replace(tagEnd, '');
+    if (savedSolution === 'savedMemento()') {
+      interactionHtmlWithoutEnd += ` [saved-solution]="${savedSolution}"`;
```

Comment:
Why is this part modified?

Reply:
hmm, after endexploration migration, the else part is useless.
I tried running e2e five times and it did not cause any issues.

(When I look at the frontend test coverage report, I notice that else part is no longer in use, which means it isn't being invoked and tested.)

Reply:
Sorry @vojtechjelinek , can you review [this PR](https://github.com/oppia/oppia/pull/14766).

[as per this discussion](https://github.com/oppia/oppia/pull/14723#issuecomment-1020881699)